app/api-fetch/all-artists.ts

```
@@ -21,12 +22,25 @@ export default async function fetchArtists() {
         throw new Error('API URL not set');
     }
     try {
-        const res = await axios.get(`${process.env.NEXT_PUBLIC_API_URL}/v1/artist`);
-        // parse to json
-        const data = ArtistSchema.parse(res.data);
-        return data["data"] as Artist[];
+        if (localStorage.getItem("artists") !== null || Date.now() - parseInt(localStorage.getItem("artistsTime")!) < 3600000) {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)